### PR TITLE
Set precise on all instructions.

### PR DESCRIPTION
**Pull request by python3kgae**

### CODE REVIEW

File: lib/HLSL/DxilGenerationPass.cpp [hunk 1]
```diff
@@ -1364,9 +1364,8 @@ static void PropagatePreciseAttributeOnOperand(Value *V, DxilTypeSystem &typeSys
     return;
   // TODO: skip precise on integer type, sample instruction...
 
-  // Set precise fast math on those instructions that support it.
-  if (DxilModule::PreservesFastMathFlags(I))
```
Comment (dmpots) [inline lib/HLSL/DxilGenerationPass.cpp]: We added this because we can’t round trip the assembly if we set fast math everywhere. I think we need it

Comment (python3kgae) [reply lib/HLSL/DxilGenerationPass.cpp]: Here what we're doing is remove the fast math. Not add it.

Comment (dmpots) [reply lib/HLSL/DxilGenerationPass.cpp]: But if the instruction does not preserve precise then why are we setting it here? Does that function have a bug?

Comment (dmpots) [reply lib/HLSL/DxilGenerationPass.cpp]: But if the instruction does not preserve precise then why are we setting it here? Does that function have a bug?

**Pull request closed without merging**